src/MicrosoftTeams.schema.json

```diff
@@ -157,7 +160,7 @@
         },
         "staticTabs": {
             "type": "array",
-            "description": "A set of tabs that may be 'pinned' by default, without the user adding them manually.  Static tabs declared in personal scope are always pinned to the app's personal experience.  Static tabs declared in the team scope are candidates to be pinned automatically to channels, subject to approval.",
+            "description": "A set of tabs that may be 'pinned' by default, without the user adding them manually.  Static tabs declared in personal scope are always pinned to the app's personal experience.  Static tabs declared in the team scope are candidates to be pinned automatically to channels, subject to a user's approval.",
```

Comment:
OK, makes sense, but what should we say instead of "a user's approval"? Simply saying "approval" isn't terribly helpful: whose approval?